source/ui/Components/Settings/default-settings.json

```diff
@@ -1,4 +1,6 @@
-{
+{
     "snapshotLocation": null,
-    "theme": "default"
+    "theme": "default",
+    "removeSnapshotsPermanently": null,

```

Comment:
I think `removeSnapshotsPermanently` and `overrideSnapshots` should default to `false` instead of `null`, otherwise you have unsaved settings at the beginning.